File: C programing/Assignment5/Ftype4/digit.c
//using type 4

#include<stdio.h>
int digit(int);
void main(){
    int a;
    printf("enter the number");
    scanf("%d",&a);
    int b=digit(a);
    printf("%d",b);
}
int digit(int a){
    int i;
    int sum;
    int r1=a%10;
    while(a>10){
        a=a/10;
    }
    sum=r1+a;
    return sum;
}

File: core java/src/Digit.java

class Digit {

	public static void main(String[] args) {
		int n=20;
	    System.out.println("enter the number");

	    int i;
	    int sum;
	    int r=n%10;

	    while(n>10){
	        n=n/10;
	    }
	    sum=r+n;
	    System.out.println("the sum of number is "+sum);

	}

}
